package.json

```diff
@@ -32,8 +33,9 @@
     "karma-coverage": "^1.0.0",
     "karma-jasmine": "^0.3.6",
     "karma-phantomjs-launcher": "^0.2.1",
+    "moment": "^2.17.1",
     "phantomjs": "^1.9.18",
-    "protractor": "^3.1.1",
+    "protractor": "^4.0.8",
```

Comment:
I agree. With this new version (4.0.8) you still need to start the webdriver manually, but at least it runs on Chrome, haha